project/make.sh

```diff
@@ -118,7 +118,8 @@ EXTLDFLAGS_STATIC='-static'
 # with options like -race.
 ORIG_BUILDFLAGS=( -a -tags "netgo static_build $DOCKER_BUILDTAGS" -installsuffix netgo )
 # see https://github.com/golang/go/issues/9369#issuecomment-69864440 for why -installsuffix is necessary here
-BUILDFLAGS=( $BUILDFLAGS "${ORIG_BUILDFLAGS[@]}" )
+eval BUILDFLAGS=( "$BUILDFLAGS" )
```

Comment:
Honestly speaking I also do not like `eval`.I will try another way while I have no idea for now